pkg/lib/cockpit.js

```diff
@@ -4454,7 +4454,7 @@
                 mapping = { };
                 meta.metrics.forEach(function(metric, i) {
                     const map = { "": i };
-                    const name = options.metrics_path_names ? options.metrics_path_names[i] : metric.name;
+                    const name = options.metrics_path_names?.[i] ?? metric.name;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18448-20230312-091152-46c0aa82-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)